app/components/calls_to_action/adviser_block_component.rb

```diff
@@ -0,0 +1,22 @@
+module CallsToAction
+  class AdviserBlockComponent < ViewComponent::Base
+    attr_reader :title, :image, :title_color, :text, :link_text, :link_target
+
+    def initialize(
+      title: "Get an adviser",
```

Comment:
```suggestion
      title: "Get a free adviser",
```

Reply:
@emmaachesongray You happy if we add `free` here? We know it has a good impact elsewhere